### PR TITLE
Release longhorn v1.3.2 into Rancher 2.7

### PR DESCRIPTION
**Pull request by PhanLe1010**
https://github.com/rancher/rancher/issues/39224

### CODE REVIEW

File: packages/longhorn/longhorn-1.3/generated-changes/patch/Chart.yaml.patch [hunk 1]
```diff
@@ -9,11 +9,11 @@
 +  catalog.cattle.io/namespace: longhorn-system
 +  catalog.cattle.io/os: linux
 +  catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
-+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 < 2.7.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
```
Comment (PhanLe1010) [inline packages/longhorn/longhorn-1.3/generated-changes/patch/Chart.yaml.patch:12]: How do you think @aiyengar2 @innobead @derekbit ?

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM, but the same question about the exclude in the crd chart. 

Also, suggest consolidate these two commits because there was one for 1.3.2-rc1.

**Comment by PhanLe1010** [comment]
> In general, LGTM, but the same question about the exclude in the crd chart.

I responded in the other thread. Thanks 

> Also, suggest consolidate these two commits because there was one for 1.3.2-rc1.

Thank you. Fixed

**Review by innobead — DISMISSED** [review]

**Comment by khushboo-rancher** [comment]
Validated on Rancher 2.7.0-rc7, Installation, uninstallation and some P0 checks are working fine.

**Comment by khushboo-rancher** [comment]
I verified the upgrade path as below, LGTM.
Longhorn v1.3.1 on Rancher 2.6.9 --> Rancher 2.7.0 --> Longhorn 1.3.2

**Comment by PhanLe1010** [comment]
@aiyengar2 @PennyScissors Can we move this one forward?

**Comment by PennyScissors** [comment]
Please split this into 2 separate commits. First should be changes made packages and release.yaml, and the second should be the output of running make charts.

**Review by pennyscissors — CHANGES_REQUESTED** [review]
Please split this into 2 separate commits. First should be changes made packages and release.yaml, and the second should be the output of running make charts.

**Comment by PhanLe1010** [comment]
> Please split this into 2 separate commits. First should be changes made packages and release.yaml, and the second should be the output of running make charts.

@PennyScissors Done. Thank you

**Comment by yangchiu** [comment]
Test installallation and uninstallation on rancher v2.7.0: **Pass**

File: packages/longhorn/longhorn-1.3/generated-changes/patch/Chart.yaml.patch (hunk 1 above)
Comment (pennyscissors) [reply packages/longhorn/longhorn-1.3/generated-changes/patch/Chart.yaml.patch:12]: lgtm

**Review by pennyscissors — APPROVED** [review]

**Pull request merged**